client/status/status.js

```diff
@@ -178,20 +201,48 @@ fetchJson('config').then(config => {
   document.forms[0].config.value = JSON.stringify(
       config.current, null, '  ');
 
+  let modulesInPlaylist = buildPlayableModuleMap(config);
+
   // Draw the list of available modules in the "play immediately" section.
   let module_list = document.getElementById('module_list');
   config.current.modules.forEach(function(element) {
     let li = document.createElement("li");
     let a = document.createElement("a");
     a.id = 'module_' + element.name;
-    a.addEventListener('click', function() {
-      fetch('/api/play?module=' + element.name,
-            {method: 'POST', credentials: 'same-origin'});
-    });
+    if (element.name in modulesInPlaylist) {
+      a.addEventListener('click', function() {
+        fetch('/api/play?module=' + element.name, { method: 'POST' });
+      });
+      a.style.textDecoration = 'underline blue';
```

Comment:
Rather than adding styling imperatively, why not add standard styles to index.html.  Something like:

#module_list a {
  text-decoration: underline blue;
  cursor: pointer;
  margin-right: 30px;
}

Then you can also eliminate all the additional "spacer" elements.